### PR TITLE
Fixes #4018, Fixes #4019. Updated Description and ComputerName parameter for Get-EventLog.

### LINKED ISSUE #4018: Content says Get-EventLog supports value 'localhost' for -ComputerName parameter; this is actually not the case
Hi,

I am trying to use the cmdlet ```Get-EventLog```.  The docs (at: https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-eventlog?view=powershell-5.1) say that the ```-ComputerName``` parameter accepts as input the value ```localhost``` to mean the local computer.  I know that I can also not supply the parameter, or put ```.```, but this is not possible as this is part of a larger script I am working on for my customer and they want the option to pass the computer name they want events pulled from, so I am thinking I am going to have to read their choice in from a param to my script and then pass -ComputerName $param1 to the ```Get-EventLog``` cmdlet.

But my issue is, your docs have an errata.  They specifically say ```localhost``` is a valid value for ```Get-EventLog```; however, as can be seen by the PowerShell output below, this is not the case:
```
PS > $PSVersionTable.PSVersion

Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      17763  316


PS > Get-EventLog -LogName Security -ComputerName localhost
Get-EventLog : The network path was not found.
At line:1 char:1
+ Get-EventLog -LogName Security -ComputerName localhost
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Get-EventLog], IOException
    + FullyQualifiedErrorId : System.IO.IOException,Microsoft.PowerShell.Commands.GetEventLogCommand
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bf280305-63b5-a41b-fe39-4c50484029c6
* Version Independent ID: 8fd10e62-dee2-56fd-e3c1-07c07f8854a3
* Content: [Get-EventLog (Microsoft.PowerShell.Management)](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-eventlog?view=powershell-5.1)
* Content Source: [reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md)
* Product: **powershell**
* GitHub Login: @daxian-dbw
* Microsoft Alias: **dongbow**

### LINKED ISSUE #4019: Get-EventLog is stated by docs to accept IP addresses for the -ComputerName parameter; this functionality does not actually work
This doc says ```Get-EventLog``` takes IP addresses for the ```-ComputerName``` parameter, however, at least it's true that the "loopback address," i.e., ```127.0.0.1``` which is well-known, does not work, as is shown in the PowerShell session below.

I have no way of testing, on the computer I am currently working on, whether other IP addresses work, but I am surprised that ```127.0.0.1``` is not accepted.  Either the IP-address functionality is broken completely in this cmdlet and the docs are wrong, or it just won't take the Loopback address (which it should, to mean the same computer it's being run on), or the docs are plain wrong in that IP addresses just are not accepted by the ```-ComputerName``` parameter.
```
PS > $PSVersionTable.PSVersion

Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      17763  316


PS > Get-EventLog -LogName Security -ComputerName "127.0.0.1"
Get-EventLog : The network path was not found.
At line:1 char:1
+ Get-EventLog -LogName Security -ComputerName "127.0.0.1"
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Get-EventLog], IOException
    + FullyQualifiedErrorId : System.IO.IOException,Microsoft.PowerShell.Commands.GetEventLogCommand
```
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bf280305-63b5-a41b-fe39-4c50484029c6
* Version Independent ID: 8fd10e62-dee2-56fd-e3c1-07c07f8854a3
* Content: [Get-EventLog (Microsoft.PowerShell.Management)](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-eventlog?view=powershell-5.1)
* Content Source: [reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md)
* Product: **powershell**
* GitHub Login: @daxian-dbw
* Microsoft Alias: **dongbow**

### PR DESCRIPTION
**Pull request by davidsmatlak**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [ ] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Acrolinx scores
Original version: 94
Updated version: 94

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[fcda641](https://github.com/davidsmatlak/PowerShell-Docs/pull/4059/commits/fcda6414fed393dc825f4d802ddd262866bcda1a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue4018_4019/reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-3.0&branch=pr-en-us-4059) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue4018_4019/reference/4.0/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-4.0&branch=pr-en-us-4059) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue4018_4019/reference/5.0/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-5.0&branch=pr-en-us-4059) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue4018_4019/reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-5.1&branch=pr-en-us-4059) (powershell-5.1) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C26%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903261735548203-4059%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=BfaDczcGeJ9HYufioyT5qJMgeR3KcA07VGjEsvpMio4%3D&st=2019-03-26T17%3A35%3A54Z&se=2019-04-26T17%3A40%3A54Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md [hunk 1]
```diff
@@ -373,15 +376,17 @@ Accept wildcard characters: False
...
 
-Specifies a remote computer. If a **ComputerName** is not specified, `Get-EventLog` uses the local
-computer.
+This parameter specifies a remote computer's NetBIOS name, Internet Protocol (IP) address, or a
+fully qualified domain name (FQDN).
 
-Type the NetBIOS name, an Internet Protocol (IP) address, or a fully qualified domain name (FQDN)
-of a remote computer. To specify the local computer, type the computer name, a dot (`.`), or
-localhost.
+If the **ComputerName** parameter isn't specified, `Get-EventLog` uses the default, which is the
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md]: Change this to:

> If the **ComputerName** parameter isn't specified, `Get-EventLog` defaults to the local
> computer. The parameter also accepts a dot (`.`) to specify the local computer.

Don't mention "localhost"

File: reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md [hunk 2]
```diff
@@ -390,7 +395,7 @@ Aliases: Cn
 
 Required: False
 Position: Named
-Default value: None
+Default value: localhost
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md]: Change this to "None"

File: reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md [hunk 3]
```diff
@@ -389,7 +393,7 @@ Aliases: Cn
 
 Required: False
 Position: Named
-Default value: None
+Default value: localhost
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md]: Change this to "None"

File: reference/4.0/Microsoft.PowerShell.Management/Get-EventLog.md [hunk 4]
```diff
@@ -389,7 +393,7 @@ Aliases: Cn
 
 Required: False
 Position: Named
-Default value: None
+Default value: localhost
```
Comment (sdwheeler) [inline reference/4.0/Microsoft.PowerShell.Management/Get-EventLog.md]: Change this to "None"

File: reference/5.0/Microsoft.PowerShell.Management/Get-EventLog.md [hunk 5]
```diff
@@ -388,7 +393,7 @@ Aliases: Cn
 
 Required: False
 Position: Named
-Default value: None
+Default value: localhost
```
Comment (sdwheeler) [inline reference/5.0/Microsoft.PowerShell.Management/Get-EventLog.md]: Change this to "None"

File: reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md (hunk 1 above)
Comment (davidsmatlak) [reply reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md]: Resolved, thanks for the feedback.

File: reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md (hunk 2 above)
Comment (davidsmatlak) [reply reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md]: Resolved, thanks for the feedback.

File: reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md (hunk 3 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md]: Resolved, thanks for the feedback.

File: reference/4.0/Microsoft.PowerShell.Management/Get-EventLog.md (hunk 4 above)
Comment (davidsmatlak) [reply reference/4.0/Microsoft.PowerShell.Management/Get-EventLog.md]: Resolved, thanks for the feedback.

File: reference/5.0/Microsoft.PowerShell.Management/Get-EventLog.md (hunk 5 above)
Comment (davidsmatlak) [reply reference/5.0/Microsoft.PowerShell.Management/Get-EventLog.md]: Resolved, thanks for the feedback.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[4965e6f](https://github.com/davidsmatlak/PowerShell-Docs/pull/4059/commits/4965e6fec8a3156e036f9d11291697a4eee891ae)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue4018_4019/reference/3.0/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-3.0&branch=pr-en-us-4059) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue4018_4019/reference/4.0/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-4.0&branch=pr-en-us-4059) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue4018_4019/reference/5.0/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-5.0&branch=pr-en-us-4059) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue4018_4019/reference/5.1/Microsoft.PowerShell.Management/Get-EventLog.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-eventlog?view=powershell-5.1&branch=pr-en-us-4059) (powershell-5.1) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C26%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903262110361955-4059%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=57MRZlfQpsPVrh4FoooRr%2BjWmdsaJrwyLl82xcuZ4QQ%3D&st=2019-03-26T21%3A10%3A38Z&se=2019-04-26T21%3A15%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**